app/models/miq_request.rb

```diff
@@ -456,6 +459,8 @@ def create_request_tasks
       request_task_created = 0
       requested_tasks.each do |idx|
         req_task = create_request_task(idx)
+        next unless req_task
```

Comment:
As discussed, we should avoid this and instead look at overriding `requested_task_idx` to define a single task regardless of the number of systems being updated.

There are a few examples in the code base of this being done: https://github.com/ManageIQ/manageiq/blob/master/app/models/automation_request.rb#L68-L70

Reply:
Thanks, now it feels much less hacky.